app/classes/Transvision/ShowResults.php

```diff
@@ -508,4 +500,45 @@ public static function searchEntities($source_strings, $regex)
 
         return $entities;
     }
+
+    /**
+     * Build the error message for target string. Can return a combination of
+     * errors, including missing final dot, long string and short string.
+     *
+     * @param string $source_string String from the source locale
+     * @param string $target_string String from the target locale
+     *
+     * @return string A concatenated string of errors or an empty string if
+     *                there is no error.
+     */
+    public static function buildErrorString($source_string, $target_string)
+    {
+        $error_message = '';
+
+        // Check for final dot
+        if (substr(strip_tags($source_string), -1) == '.'
+            && substr(strip_tags($target_string), -1) != '.') {
+            $error_message = '<em class="error"> No final dot?</em>';
```

Comment:
I would really put the space after the `</em>` in all messages, not at the beginning of the label.